Automattic/woocommerce-payments
# Remove unsupported EUR currency from Afterpay payment method

Fixes #7713

ES and FR were part of the initial scope for Afterpay, but plans changed. We already removed support for those countries  on the server side https://github.com/Automattic/woocommerce-payments-server/pull/3562 and we want to remove the unsupported currency from the client.

#### Changes proposed in this Pull Request
* Remove unsupported EUR currency from Afterpay payment method.








#### Testing instructions





* Sanity check that checkout flow with Afterpay is not affected.

-------------------


**Post merge**
